src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/ContentTypeExtensions.cs

```diff
@@ -81,5 +81,10 @@ public static bool IsKeyVaultReference(this ContentType contentType)
         {
             return contentType.MediaType.Equals(KeyVaultConstants.ContentType);
         }
+
+        public static bool IsSnapshotReference(this ContentType contentType)
+        {
+            return contentType.MediaType.Equals(SnapshotReferenceConstants.ContentType);
```

Comment:
It's a bit odd that `IsAi` and `IsAiChatCompletion` above are more complex. Did you verify this works as expected?

Reply:
I just realized that we only match the profile for AI and AIChatCompletion content types. But we match the entire content type for feature flags and key vault refs. Is that because we want to be more relaxed for AI content type tracking?

Reply:
> Did you verify this works as expected?

?

Reply:
Or are you saying you will be removing this function?